liveblog/liveblog
# [LBSD-2882] - Create custom voting endpoint for polls

### Purpose

This PR implements a custom voting endpoint for polls to allow for better poll voting management 

### What has changed

- Added new `client_poll_vote` endpoint to be used to place voted from the embed
- Changed to use of $inc function for incremental voting
- Changed the endpoint called on the embeds
- Added tests for the new endpoint

### Steps to test

1. Open the Editor and publish a new poll.
2. Open the preview on multiple browsers and place your votes.
3. Observe that the request to send the vote is a new endpoint

Resolves: [LBSD-2882](https://sofab.atlassian.net/browse/LBSD-2882)

[LBSD-2882]: https://sofab.atlassian.net/browse/LBSD-2882?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
